### PR TITLE
Update ftl pageBuilder 

### PR DESCRIPTION
**Pull request by jvega190**

### CODE REVIEW

File: craftercms-plugin.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ plugin:
   version:
```
[the remark this answers is not in the document]
Comment (jvega190) [reply craftercms-plugin.yaml:15]: Older versions already removed

File: config/studio/content-types/component/post/form-definition.xml [hunk 2]
```diff
@@ -542,6 +542,11 @@
 							<value />
 							<type>boolean</type>
 						</property>
+						<property>
+							<name>contentTypes</name>
+							<value>,/component/bio</value>
```
[the remark this answers is not in the document]
Comment (jvega190) [reply config/studio/content-types/component/post/form-definition.xml]: It was due to outdated files, re-added the control and now the leading commas are not there

**Comment by jvega190** [comment]
Screenshot for this ticket https://github.com/craftercms/craftercms/issues/4671 is in this PR

File: config/studio/ui.xml [hunk 3]
```diff
@@ -0,0 +1,430 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (rart) [inline config/studio/ui.xml:1]: Outdated xml

**Review by rart — CHANGES_REQUESTED** [review]

**Comment by jvega190** [comment]
@rart updates done in here https://github.com/craftercms/wordify-blueprint/pull/39 moved to this PR, and reviews for that PR are done in here too

File: templates/web/fragments/footer.ftl [hunk 4]
```diff
@@ -1,36 +1,51 @@
-<#import "/templates/system/common/cstudio-support.ftl" as studio />
+<#import "/templates/system/common/crafter.ftl" as crafter />
 
-<footer class="site-footer" <@studio.componentAttr component=contentModel ice=true />>
+<@crafter.tag $tag="footer" $model=contentModel class="site-footer">
```
Comment (rart) [inline templates/web/fragments/footer.ftl:3]: crafter.tag

**Review by rart — CHANGES_REQUESTED** [review]

**Review by rart — APPROVED** [review]

**Comment by sumerjabri** [comment]
@rart why are we moving away from the `crafter` namespace?

**Comment by rart** [comment]
@jvega190 what we discussed was extracting navigation out of ice.ftl, not out of crafter.ftl

fyi @sumerjabri

**Review by rart — APPROVED** [review]

**Pull request merged**